sig-etcd/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - sig-etcd-leads
+approvers:
+  - sig-etcd-leads
+labels:
+  - sig/etcd
```

Comment:
Is this label ready?

> @pacoxu: The label(s) sig/etcd cannot be applied, because the repository doesn't have them.

https://github.com/kubernetes/kubernetes/pull/118077#issuecomment-1714982898

Reply:
Thanks for reminding, this is done now: https://github.com/kubernetes/test-infra/pull/30948